components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -337,7 +337,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
             return;
         }
 
-        if (workspaceInstance.status.phase === "preparing") {
+        if (workspaceInstance.status.phase === "building") {
```

Comment:
This should be the case for both "preparing" and "building".

Reply:
cad708c11